ldk/javascript/examples/self-test-loop/src/tests/screen/index.ts

```diff
@@ -73,7 +108,24 @@ const rebuildImage = (results: screen.OCRResult[]) => {
qO9sEzBCAAgRkC2ffQTD6xEBgR4DeAI0L4jyNQvYxV43RAvnSVBs8QgMBJBK58p53UP72cRYD/Cvis/aCbCwT00rbiC9L1VmhdGIcUCEAAAv9LgO8iDsJOAlwAd9JFGwIQgAAEIAABCBxIgD8BH7gptAQBCEAAAhCAAAR2EuACuJMu2hCAAAQgAAEIQOBAAlwAD9wUWoIABCAAAQhAAAI7CXAB3EkXbQhAAAIQgAAEIHAgAS6AB24KLUEAAhCAAAQgAIGdBLgA7qSLNgQgAAEIQAACEDiQABfAAzeFliAAAQhAAAIQgMBOAlwAd9JFGwIQgAAEIAABCBxIgAvggZtCSxCAAAQgAAEIQGAnAS6AO+miDQEIQAACEIAABA4kwAXwwE2hJQhAAAIQgAAEILCTABfAnXTRhgAEIAABCEAAAgcS4AJ44KbQEgQgAAEIQAACENhJgAvgTrpoQwACEIAABCAAgQMJcAE8cFNoCQIQgAAEIAABCOwkwAVwJ120IQABCEAAAhCAwIEEuAAeuCm0BAEIQAACEIAABHYS4AK4ky7aEIAABCAAAQhA4EACXAAP3BRaggAEIAABCEAAAjsJcAHcSRdtCEAAAhCAAAQgcCABLoAHbgotQQACEIAABCAAgZ0EuADupIs2BCAAAQhAAAIQOJAAF8ADN4WWIAABCEAAAhCAwE4CXAB30kUbAhCAAAQgAAEIHEiAC+CBm0JLEIAABCAAAQhAYCcBLoA76aINAQhAAAIQgAAEDiTABfDATaElCEAAAhCAAAQgsJMAF8CddNGGAAQgAAEIQAACBxLgAnjgptASBCAAAQhAAAIQ2EmAC+BOumhDAAIQgAAEIACBAwlwATxwU2gJAhCAAAQgAAEI7CTABXAnXbQhAAEIQAACEIDAgQS4AB64KbQEAQhAAAIQgAAEdhLgAriTLtoQgAAEIAABCEDgQAJcAA/cFFqCAAQgAAEIQAACOwlwAdxJF20IQAACEIAABCBwIAEugAduCi1BAAIQgAAEIACBnQS4AO6kizYEIAABCEAAAhA4kAAXwAM3hZYgAAEIQAACEIDATgJcAHfSRRsCEIAABCAAAQgcSIAL4IGbQksQgAAEIAABCEBgJwEugDvpog0BCEAAAhCAAAQOJMAF8MBNoSUIQAACEIAABCCwkwAXwJ100YYABCAAAQhAAAIHEuACeOCm0BIEIAABCEAAAhDYSYAL4E66aEMAAhCAAAQgAIEDCXABPHBTaAkCEIAABCAAAQjsJMAFcCddtCEAAQhAAAIQgMCBBLgAHrgptAQBCEAAAhCAAAR2EuACuJMu2hCAAAQgAAEIQOBAAlwAD9wUWoIABCAAAQhAAAI7CXAB3EkXbQhAAAIQgAAEIHAgAS6AB24KLUEAAhCAAAQgAIGdBLgA7qSLNgQgAAEIQAACEDiQABfAAzeFliAAAQhAAAIQgMBOAlwAd9JFGwIQgAAEIAABCBxIgAvggZtCSxCAAAQgAAEIQGAnAS6AO+miDQEIQAACEIAABA4kwAXwwE2hJQhAAAIQgAAEILCTABfAnXTRhgAEIAABCEAAAgcS4AJ44KbQEgQgAAEIQAACENhJgAvgTrpoQwACEIAABCAAgQMJcAE8cFNoCQIQgAAEIAABCOwkwAVwJ120IQABCEAAAhCAwIEEuAAeuCm0BAEIQAACEIAABHYS4AK4ky7aEIAABCAAAQhA4EACXAAP3BRaggAEIAABCEAAAjsJcAHcSRdtCEAAAhCAAAQgcCABLoAHbgotQQACEIAABCAAgZ0EuADupIs2BCAAAQhAAAIQOJAAF8ADN4WWIAABCEAAAhCAwE4CXAB30kUbAhCAAAQgAAEIHEiAC+CBm0JLEIAABCAAAQhAYCcBLoA76aINAQhAAAIQgAAEDiTABfDATaElCEAAAhCAAAQgsJMAF8CddNGGAAQgAAEIQAACBxLgAnjgptASBCAAAQhAAAIQ2EmAC+BOumhDAAIQgAAEIACBAwlwATxwU2gJAhCAAAQgAAEI7CTABXAnXbQhAAEIQAACEIDAgQS4AB64KbQEAQhAAAIQgAAEdhLgAriTLtoQgAAEIAABCEDgQAJcAA/cFFqCAAQgAAEIQAACOwlwAdxJF20IQAACEIAABCBwIAEugAduCi1BAAIQgAAEIACBnQS4AO6kizYEIAABCEAAAhA4kAAXwAM3hZYgAAEIQAACEIDATgJcAHfSRRsCEIAABCAAAQgcSIAL4IGbQksQgAAEIAABCEBgJwEugDvpog0BCEAAAhCAAAQOJMAF8MBNoSUIQAACEIAABCCwkwAXwJ100YYABCAAAQhAAAIHEuACeOCm0BIEIAABCEAAAhDYSYAL4E66aEMAAhCAAAQgAIEDCXABPHBTaAkCEIAABCAAAQjsJMAFcCddtCEAAQhAAAIQgMCBBLgAHrgptAQBCEAAAhCAAAR2EuACuJMu2hCAAAQgAAEIQOBAAlwAD9wUWoIABCAAAQhAAAI7CXAB3EkXbQhAAAIQgAAEIHAgAS6AB24KLUEAAhCAAAQgAIGdBLgA7qSLNgQgAAEIQAACEDiQABfAAzeFliAAAQhAAAIQgMBOAlwAd9JFGwIQgAAEIAABCBxIgAvggZtCSxCAAAQgAAEIQGAnAS6AO+miDQEIQAACEIAABA4kwAXwwE2hJQhAAAIQgAAEILCTABfAnXTRhgAEIAABCEAAAgcS4AJ44KbQEgQgAAEIQAACENhJgAvgTrpoQwACEIAABCAAgQMJcAE8cFNoCQIQgAAEIAABCOwkwAVwJ120IQABCEAAAhCAwIEEuAAeuCm0BAEIQAACEIAABHYS4AK4ky7aEIAABCAAAQhA4EACXAAP3BRaggAEIAABCEAAAjsJcAHcSRdtCEAAAhCAAAQgcCABLoAHbgotQQACEIAABCAAgZ0EuADupIs2BCAAAQhAAAIQOJDA/wd1oDSPFnUzlQAAAABJRU5ErkJggg==';
```

Comment:
It will be better to read the image and convert the image to base64 string here.
you can do something like this:
```let request = await network.httpRequest({
    url: 'https://github.com/open-olive/loop-development-kit/raw/develop/ldk/javascript/examples/self-test-loop/static/testocr.png',
    method: 'GET',
  });
  let branch = 'develop';

  // Above URL is used after this feature is finished and merged in
  // Before PR is merged and branch is deleted, use the feature branch
  // Can delete this block after merge
  if (request.statusCode === 404) {
    request = await network.httpRequest({
      url: 'https://github.com/open-olive/loop-development-kit/raw/HELPS-3796-ocrFileEncoded/ldk/javascript/examples/self-test-loop/static/testocr.png',
      method: 'GET',
    });
    branch = 'HELPS-3796-ocrFileEncoded';
  }

  const encodedImage = Buffer.from(request.body, "base64").toString();